src/js/pages/home/LogoTitle.js

```diff
@@ -8,7 +8,7 @@ class LogoTitle extends Component {
 			<div>
 				<Image fluid src={trap} />
 				<div className="pageTitle">
-					<font size="50">Translation made available </font>
+					<font size="50">{this.props.text} </font>
```

Comment:
text should probably be renamed to title_text or something similar, in the event that more text is added later?